docs/src/pages/[platform]/connected-components/authenticator/customization/customization.headers-and-footers.web.mdx

```diff
@@ -37,15 +37,16 @@ import { Component } from '@angular/core';
 import { Amplify } from 'aws-amplify';
 import { AuthenticatorService } from '@aws-amplify/ui-angular';
 
-import awsExports from './aws-exports';
+// import config from './aws-exports'; // Amplify Gen 1 config
+import config from './amplify_outputs.json'
```

Comment:
Could we just have a single example block in "getting started" with that handles the "Gen1/Gen2" import treatment and reference code snippets from the relevant example apps, and add `IGNORE` statements for the constructor, e.g.:
```
export class UseAuthenticatorComponent {
   // IGNORE
  constructor(public authenticator: AuthenticatorService) {
    // IGNORE
    Amplify.configure(config);
     // IGNORE
  }
}
```

Reply:
this is in the docs, so if we can document it once on the Angular Authenticator pages separately, we could just delete this code

Reply:
Agree with deleting here. Should have explained though, my thoughts for "IGNORE" were for the examples where code needs to be added directly in the component classes, like the "form fields" example